hadoop-hdds/tools/src/main/java/org/apache/hadoop/hdds/scm/cli/container/InfoSubcommand.java

```diff
@@ -109,6 +113,21 @@ public void execute(ScmClient scmClient) throws IOException {
       } else {
         LOG.info("Pipeline id: {}", container.getPipeline().getId().getId());
       }
+      LOG.info("Write PipelineId: {}",
```

Comment:
Already logging pipeline Id in if/else block, this is redundant, above can be modified if requried.

Reply:
Both are different, in else we have Pipeline Id(container.getPipeline().getId().getId()) and below we have Write Pipeline Id(container.getContainerInfo().getPipelineID().getId()).

Reply:
@sumitagrawl "Write Pipeline ID" represents original pipeline on which the container was created whereas once pipeline is closed we use random pipeline for read. These two values will be the same till pipeline is closed.